documentation/assemblies/deploying/assembly-deploy-kafka-connect.adoc

```diff
@@ -8,6 +8,9 @@
 
 link:https://kafka.apache.org/documentation/#connect[Kafka Connect^] is a tool for streaming data between Apache Kafka and external systems.
 
+In Strimzi, Kafka Connect is deployed in distributed mode.
+Kafka Connect can also work in standalone mode, but this is currently not supported by Strimzi.
```

Comment:
using "currently" let me think that it could be in the future. I would remove it, I don't think we are going to support it.

Reply:
I guess I wrote it like this. I think we have it somewhere at the bottom of our backlog. There are lot of requirements for Connect to work with PVs etc. and standalone as a _singleton_ might be easier that way. That said, I do not plan any work on that anytime soon. So I can live without the `currently` as well.

Reply:
I removed _currently_